submissions/SerhiiYakovenko/memory-pair-game/script.js

```diff
@@ -0,0 +1,128 @@
+const cards = [
+  {
+    name: "alt",
+    img: "img/alt.png",
+  },
+  {
+    name: "hanako",
+    img: "img/hanako.png",
+  },
+  {
+    name: "kerry_eurodyne",
+    img: "img/kerry_eurodyne.png",
+  },
+  {
+    name: "saburo",
+    img: "img/saburo.png",
+  },
+  {
+    name: "silverhand",
+    img: "img/silverhand.png",
+  },
+  {
+    name: "smasher",
+    img: "img/smasher.png",
+  },
+  {
+    name: "v-female",
+    img: "img/v-female.png",
+  },
+  {
+    name: "v-male",
+    img: "img/v-male.png",
+  },
+];
+
+const div = document.createElement("div");
+div.id = "board";
+document.body.appendChild(div);
+
+let mixedCards = cards.concat(cards).sort(function () {
+  return 0.5 - Math.random();
+});
+
+let firstFlip = "";
+let secondFlip = "";
+let turn = 0;
+let previousCard = null;
+
+const board = document.getElementById("board");
+const boardGrid = document.createElement("section");
+boardGrid.setAttribute("class", "boardGrid");
```

Comment:
Using of classList is more flexible, so try to use [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) for styling